src/cable/connection.cr

```diff
@@ -4,81 +4,56 @@ module Cable
   class Connection
     class UnathorizedConnectionException < Exception; end
 
-    @@mock : Cable::Connection?
-
     property internal_identifier : String = "0"
+    property connection_identifier : String = ""
+
     getter token : String?
-    getter connection_identifier : String
+    getter reject_connection : Bool = false
```

Comment:
```suggestion
    getter? connection_rejected : Bool = false
```

If we do this, then we won't need the method below

Reply:
great point!